### ISSUE TITLE
file.managed backup: minion seems not to work

### ISSUE THREAD

**Issue by m7v8** [saltstack/salt#27787]
After running:
`salt-ssh <host> state.sls database`

With this state defintion (shortened):
database/init.sls:

```
postgresql:                             
  [..]
  file.managed:
    - name: {{ salt['user.info']('postgres').home }}/data/postgresql.conf
    - source: salt://database/files/postgresql.conf
    - backup: minion
    - template: jinja
    - require:
      - pkg: postgresql
```

And of course changes in 'database/files/postgresql.conf' salt will not generate a backup file under: /var/cache/salt/minion/
I currently can not verify if this only happens when used with salt-ssh.

Saltversion:
salt-ssh 2015.8.0 (Beryllium)
OS:
OpenSuSE 12.3

**Comment by jfindlay** [comment]
@m7v8, thanks for the report.

**Comment by MatthiasKuehneEllerhold** [comment]
Still happening with salt-ssh 2016.11.4 (Carbon) under Debian Jessie.

**Comment by grichmond-salt** [comment]
Still happening with salt-ssh 2017.7.1 on RHEL 7.4.

**Comment by dseomn** [comment]
I'm still seeing this issue with salt-ssh 3002.2.

**Comment by dseomn** [comment]
Oops, sorry, I'd updated the config file in the wrong place. Looks like this is fixed as of 3002.2.
